src/main/resources/database/data.sql

```diff
@@ -87,6 +87,28 @@ INSERT INTO `shift_comment` (employee_id, c_message, c_date) VALUES (3, 'Пот
 INSERT INTO `shift_comment` (employee_id, c_message, c_date) VALUES (3, 'Пропав wi-fi на третьому поверсі', TIMESTAMPADD(DAY, -5, CURRENT_TIMESTAMP));
 INSERT INTO `shift_comment` (employee_id, c_message, c_date) VALUES (2, 'Завтра приїдуть англомовні клієнти', TIMESTAMPADD(DAY, -6, CURRENT_TIMESTAMP));
 INSERT INTO `shift_comment` (employee_id, c_message, c_date) VALUES (1, 'Закінчуються цукерки на ресепшині', TIMESTAMPADD(DAY, -6, CURRENT_TIMESTAMP));
+INSERT INTO `shift_comment` (employee_id, c_message, c_date) VALUES (2, 'Смену закрыл, потерялся ключ от 208 номера', TIMESTAMPADD(DAY, -1, CURRENT_TIMESTAMP));
+INSERT INTO `shift_comment` (employee_id, c_message, c_date) VALUES (2, 'Женщина из 305 номера потеряла собаку, белый питбуль', TIMESTAMPADD(DAY, -2, CURRENT_TIMESTAMP));
+INSERT INTO `shift_comment` (employee_id, c_message, c_date) VALUES (2, 'В 410 номере закончились напитки в минибаре', TIMESTAMPADD(DAY, -2, CURRENT_TIMESTAMP));
+INSERT INTO `shift_comment` (employee_id, c_message, c_date) VALUES (2, 'Вышел из строя холодильник в 503 номере', TIMESTAMPADD(DAY, -3, CURRENT_TIMESTAMP));
+INSERT INTO `shift_comment` (employee_id, c_message, c_date) VALUES (2, 'Завтра приедет телевидение снимать репортаж', TIMESTAMPADD(HOUR, -6, CURRENT_TIMESTAMP));
+INSERT INTO `shift_comment` (employee_id, c_message, c_date) VALUES (2, 'На ресепшн подошли двое мужчин бандитской внешности и хотят обсудить возможность продажи отеля', TIMESTAMPADD(DAY, -6, CURRENT_TIMESTAMP));
+INSERT INTO `shift_comment` (employee_id, c_message, c_date) VALUES (1, 'Постояльцы жалуються на слабую скорость интернета', TIMESTAMPADD(MINUTE , -423, CURRENT_TIMESTAMP));
+INSERT INTO `shift_comment` (employee_id, c_message, c_date) VALUES (1, 'С Кириллом Степановичем случился несчастный случай, сегодня я его подменяю ', TIMESTAMPADD(HOUR, -6, CURRENT_TIMESTAMP));
+INSERT INTO `shift_comment` (employee_id, c_message, c_date) VALUES (1, 'Меня срочно вызвали в школу к ребёнку', TIMESTAMPADD(DAY, -4, CURRENT_TIMESTAMP));
+INSERT INTO `shift_comment` (employee_id, c_message, c_date) VALUES (1, 'Постояльцы жалуются на отсутствие горячей воды', TIMESTAMPADD(DAY, -8, CURRENT_TIMESTAMP));
+INSERT INTO `shift_comment` (employee_id, c_message, c_date) VALUES (1, 'Постояльцы продолжают жалуются на отсутствие горячей воды', TIMESTAMPADD(DAY, -7, CURRENT_TIMESTAMP));
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Define a constant instead of duplicating this literal 3 times. ([plsql:S1192](https://sonarcloud.io/organizations/final-project/rules#rule_key=plsql%3AS1192))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=final-project&pullRequest=170&issues=AWbzqUKXTJjFgJLTaxt8&open=AWbzqUKXTJjFgJLTaxt8)

[ISSUE_KEY]: AWbzqUKXTJjFgJLTaxt8